LiveSplit.TheTalosPrinciple.asl

```diff
@@ -193,10 +279,8 @@ start {
     var world = match.Groups[1].Value;
     // Main menu and settings count as 'worlds', ignore them
     if (world.Contains("Menu")) return false;
-    // Randomizer settings world
-    if (world == "Content/Talos/Levels/Randomizer/Options.wld") return false;
 
-    if (vars.cheatFlags.Current != 0) {
+    if (vars.cheatFlags != null && vars.cheatFlags.Current != 0) {
```

Comment:
Do something similar for the `isLoading {}` block?

Reply:
That one's already done like that even on master.